src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -58,12 +59,19 @@
         {
             "name": "installer",
             "defaultRemote": "https://github.com/dotnet/installer",
-            "defaultRef": "vmr"
+            "exclude": [
```

Comment:
Is there logic that will warn on invalid exclude/include paths?  I ask because eventually this content will be moved to the VMR therefore making this exclude a no-op.

Reply:
There's no logic and currently it doesn't play a big role. It's rather a sanity check for me to execute these paths when running the synchronization.

I plan to revise all of the exclusions for each repo as part of https://github.com/dotnet/arcade/issues/10263 but I might remove all of the exclusions first in order to compare the contents with the tarball first. I will keep it as-is for this PR.